src/pages/registration/RegistrationFormActions.tsx

```diff
@@ -65,7 +66,11 @@ export const RegistrationFormActions = ({
       );
       dispatch(setNotification({ message: t('feedback.success.update_registration'), variant: 'success' }));
       if (isLastTab) {
-        history.push(getRegistrationLandingPagePath(values.identifier));
+        if (history.location.state?.backPath) {
+          history.goBack();
```

Comment:
Denne er kanskje litt spesiell, siden `backPath` er hele pathname, feks `/tasks/nvi/1234`, men uansett hva som ligger der gjør vi bare en `goBack()`. Tror vi må bruke `goBack` her for at man skal få riktig location state når man går tilbake, og tror det er like greit å sette hele path i state i stedet for `canGoBack: boolean` e.l. 
Åpen for innspill her